Update CLI docs

dwlbnks opened the pull request:
Hides several CLI arguments that ReadySet does not want to consider supported at the moment. These arguments exist in the code, but will not show up if the user runs `--help`.

greg-mckeon commented on external/integrate-readyset-with-supabase.md:
```diff
@@ -0,0 +1,118 @@
+
+    1. In the command output, find the query and check the `readyset supported` value:
+
+        - If the value is `pending`, check again until you see `yes` or `no`.
```
Some may stay "pending" forever, which means "no" :)

jseldess replied:
Heard from @glittershark that this will only happen with the query hits an unreplicated table. Griffin is working on returning `no` in such cases: https://readysettech.atlassian.net/browse/ENG-2135.

jseldess commented:
@greg-mckeon, closing this PR. Please feel free to reopen and take it over, if you like.

Commit:
Update CLI docs

Hides several CLI arguments that ReadySet does not want to consider
supported at the moment. These arguments exist in the code, but will not
show up if the user runs `--help`.
